cibuildwheel/macos.py

```diff
@@ -58,14 +58,11 @@ def shell(args, env=None, cwd=None):
             before_build_prepared = prepare_command(before_build, python=python, pip=pip)
             shell(shlex.split(before_build_prepared), env=env)
 
-        # install the package first to take care of dependencies
-        shell([pip, 'install', project_dir], env=env)
-
         # build the wheel to temp dir
         temp_wheel_dir = '/tmp/tmpwheel%s' % config.version
         shell([pip, 'wheel', project_dir, '-w', temp_wheel_dir, '--no-deps'], env=env)
-        temp_wheel = glob(temp_wheel_dir+'/*.whl')[0]
 
+        temp_wheel = glob(temp_wheel_dir+'/*.whl')[0]
```

Comment:
Ok, fair enough. Fine with me.
I have just realised that "for whl in /tmp/linux_wheels/*.whl; do" is probably just easier to write the pattern match, and not per se to do that over all wheels.